ocaml/xcp-rrdd/bin/rrdp-cpu/rrdp_cpu.ml

```diff
@@ -232,14 +232,15 @@ let generate_cpu_ds_list xc () =
   let _, domains, _ = Xenctrl_lib.domain_snapshot xc in
   dss_pcpus xc @ dss_vcpus xc domains @ dss_loadavg () @ dss_hostload xc domains
 
+let max_supported_vms_per_host = 1000
```

Comment:
note that `xcp-rrdd` has `let max_supported_vms = 1024`. Should probably bring that out into the same constant somewhere and use it in both places